src/sundials.jl

```diff
@@ -194,7 +194,7 @@ function sunsim(smem::SimSundials, tstop::Float64, Nsteps::Int)
     end
     SimResult(yout, [sm.outputs[yidx]])
 end
-sunsim(ss::SimState, tstop::Float64, Nsteps::Int) = sunsim(setup_sunsim(ss,1e-4,1e-4), tstop, Nsteps)
+sunsim(ss::SimState, tstop::Float64, Nsteps::Int) = sunsim(setup_sunsim(ss,reltol=1e-4,abstol=1e-4), tstop, Nsteps)
```

Comment:
It'd be nice to also have a version of `sunsim` with reltol and abstol keyword arguments, too.